recipe/meta.yaml

```
@@ -245,6 +251,7 @@ outputs:
       run:
         - {{ pin_subpackage('_r-xgboost-mutex', exact=True) }}
         - {{ pin_subpackage('libxgboost', max_pin='x.x.x') }}
+        - libgomp  # [win]
```

Tweak selector depth to make this easier to pick out

```suggestion
        - libgomp                                                  # [linux or win]
```